htdocs/install/mysql/migration/20.0.0-21.0.0.sql

```diff
@@ -282,3 +282,5 @@ CREATE TABLE llx_product_customer_price_extrafields (
 ) ENGINE=innodb;
 
 ALTER TABLE llx_product_customer_price_extrafields ADD UNIQUE INDEX uk_product_customer_price_extrafields (fk_object);
+ALTER TABLE llx_facture ADD COLUMN payment_reference varchar(25) AFTER date_lim_reglement;
+ALTER TABLE llx_societe ADD COLUMN tp_payment_reference varchar(25) AFTER code_fournisseur;
```

Comment:
Why adding another field. Why not using the already existing field customer code ?

Reply:
Payment reference is calculated data for bank payment and sepa payments.

Reply:
Hi,
There are 2 ways of use, per invoice and per customer
In Greece per customer(we use RF) is widely used from companies issuing recurring bills e.g.: electricity, gas, telephony-internet etc
This way every customer has to know only his/her unique payment reference and amount to pay the bills each month
Per invoice is usually used in  no recurring bills
Payment reference are then included in the bank statements received by vendor who can automatically process payments - reconcile

Reply:
Some more important info...

Structured payment references like international  RF main difference with a custom reference like customer code you mentioned is that they **must** be used through your cooperating payment service provider(Banks, electronic payments providers etc).

You have to contact your bank first to let you use it(Banks need to know to whose account e.g every RF corresponds)

So you can't just generate them and use them in the payment info field like you do with the customer code, invoice ref or any other way any company uses.